src/pages/unauthenticated/PewPew.vue

```diff
@@ -723,6 +928,112 @@ export default {
+      const card = document.querySelector('div.y-translated');
+      // moves the other two cards up to simulate no change
+      const cardGroup = document.querySelectorAll('div.group-top');
+      // moves sticky card out of frame
+      const stickyCard = document.querySelector('div.move-down');
+      // moves sticky card to its original position
+      const settledSticky = document.querySelector('div.sticky-settle');
+      // moves the card group to its orginial position
+      const settledGroup = document.querySelectorAll('div.card-group');
+      // moves top card to its original position
+      const settledCard = document.querySelector('div.settle');
+
+      // set origin of transition - everything up since the new item being added
+      // is now the first item
+      if (settledCard) {
+        settledCard.classList.remove('settle');
+        settledCard.classList.add('y-translated');
+      }
+      if (settledGroup) {
+        settledGroup.forEach((group) => {
+          group.classList.remove('card-group');
+          group.classList.add('group-top');
+        });
+      }
+      if (stickyCard) {
+        stickyCard.classList.remove('move-down');
+        stickyCard.classList.add('sticky-settle');
+      }
+
+      // move everything down
+      if (card) {
+        card.classList.remove('y-translated');
+        card.classList.add('settle');
+      }
+      if (cardGroup) {
+        cardGroup.forEach((group) => {
+          group.classList.remove('group-top');
+          group.classList.add('card-group');
+        });
+      }
+      // if ten seconds have passed
+      if (settledSticky) {
+        if (Math.abs(this.startTime - new Date()) / 1000 > 10) {
+          settledSticky.classList.remove('sticky-settle');
+          settledSticky.classList.add('move-down');
+        }
+      }
+    },
+    getDateDifference(date) {
+      const dateDifference = Math.abs(new Date() - date);
+      let difference;
+      // convert it to different measurements based on how much time passed
+      if (dateDifference < 1000) {
+        difference = 'now';
+      } else {
+        difference = Math.floor(Math.abs(dateDifference / 1000));
+        if (difference >= 60) {
+          difference = Math.floor(Math.abs(difference / 60));
+          if (difference >= 60) {
+            difference = Math.floor(Math.abs(difference / 60));
+            if (difference >= 24) {
+              difference = Math.floor(Math.abs(difference / 24));
+              if (difference >= 365) {
+                difference = Math.floor(Math.abs(difference / 365));
+                difference += ' years ago';
+              } else {
+                difference += ' days ago';
+              }
+            } else {
+              difference += ' hours ago';
+            }
+          } else {
+            difference += ' minutes ago';
+          }
+        } else {
+          difference += ' seconds ago';
+        }
+      }
+      return difference;
+    },
```

Comment:
This function might not be working as you expected. You don't even have to write this function. Use `vue-moment` package that we have. 
```vue
this.$moment(myDate).fromNow()

// moment(new Date("12/10.2020")).fromNow()
// "7 months ago"

```
See: https://momentjs.com/

Reply:
done